tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = integration-tests, flake8
+skipsdist = True
+
+[testenv]
+skip_install = True
+passenv = DISPLAY MOZ_HEADLESS
+deps = -rrequirements/tests.txt
+commands = pytest --driver=Firefox --verbose {posargs:test/integration}
+
+[testenv:flake8]
+deps = 
+	-rrequirements/flake8.txt
+	-rrequirements/tests.txt
```

Comment:
You should need the test requirements for flake8.

Reply:
Do you mean shouldn't?

Reply:
I mean shouldn't. TFI Friday.

Reply:
So maybe my tox config is wrong. If I try to use isort without the other requirements installed I get errors because I think isort sees the imports as local packages not 3rd party.

Reply:
Ah, right. You can probably leave this then.